tendrl/monitoring_integration/sync/dashbaord_sync.py

```diff
@@ -18,20 +19,30 @@ def refresh_dashboard(self):
         try:
             # check alert organization is exist
             if NS.config.data["org_id"]:
-                cluster_details = {}
-                dashboards = []
-                integration_ids = utils.get_resource_keys("", "clusters")
-                for integration_id in integration_ids:
-                    key = "/clusters/%s/TendrlContext/sds_name" % \
-                        integration_id
-                    sds_name = etcd_utils.read(key).value
-                    if sds_name == constants.GLUSTER:
-                        cluster_details, dashboards = gluster_cluster_details.\
-                            get_cluster_details(
-                                integration_id
+                all_cluster_details = {}
+                integration_ids = self.get_managed_clusters_integration_id()
```

Comment:
The function `get_managed_clusters_integration_id` returns an empty list if not managed clusters, so we can safely start a for loop without having a `None` check I feel.

Reply:
ok